### PR TITLE
Select closest relevant path

### PR DESCRIPTION
**Pull request by domachine**

### CODE REVIEW

File: app/lib/app/shared/types.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export type DocumentCategory =
+  | 'csaf_base'
+  | 'csaf_security_incident_response'
+  | 'csaf_informational_advisory'
+  | 'csaf_security_advisory'
```
Comment (valentindey) [inline app/lib/app/shared/types.ts:5]: could also be `csaf_vex`

File: app/lib/app/SecvisogramPage/shared/isPropertyRelevant.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+/**
+ * @param {object} params
+ * @param {string[]} params.relevanceLevels
+ * @param {import("../../shared/types").DocumentCategory} params.category
+ * @param {import('./isPropertyRelevant/types').Property} params.property
+ * @param {string} params.selectedRelevanceLevel
+ */
+export default function ({
```
Comment (valentindey) [inline app/lib/app/SecvisogramPage/shared/isPropertyRelevant.js:8]: this function would be a nice candidate for a small unit test :)

File: app/lib/app/SecvisogramPage/View/FormEditor/editors/GenericEditor/Attributes/shared/Attribute.js [hunk 3]
```diff
@@ -51,14 +52,15 @@ export default function Attribute({
   let showAttribute = true
 
   const category = doc.document?.category
-  const relevanceLevelForCategory = category
-    ? (property.metaData?.relevanceLevels || {})[category] || ''
-    : ''
 
   if (
     category &&
```
Comment (valentindey) [inline app/lib/app/SecvisogramPage/View/FormEditor/editors/GenericEditor/Attributes/shared/Attribute.js:57]: this line seems redundant as it is also checked inside of `isPropertyRelevant`

File: app/lib/app/SecvisogramPage/View/FormEditor/editors/GenericEditor/ObjectEditor.js [hunk 4]
```diff
@@ -152,14 +153,14 @@ export default function ObjectEditor({
           </li>
         ) : null}
         {menuNodes.map((menuItem, menuItemIndex) => {
-          const relevanceLevelForCategory = category
-            ? (menuItem.property.metaData?.relevanceLevels || {})[category] ||
-              ''
-            : ''
           if (
             category &&
```
Comment (valentindey) [inline app/lib/app/SecvisogramPage/View/FormEditor/editors/GenericEditor/ObjectEditor.js:157]: same as above

**Review by valentindey — CHANGES_REQUESTED** [review]

File: app/lib/app/SecvisogramPage/shared/isPropertyRelevant.js (hunk 2 above)
Comment (domachine) [reply app/lib/app/SecvisogramPage/shared/isPropertyRelevant.js:8]: I like the e2e test ;)

**Review by cloeser — APPROVED** [review]

**Pull request merged**